python_anvil/api_resources/mutations/create_etch_packet.py

```diff
@@ -206,7 +202,43 @@ def get_file_payloads(self):
                 )
         return self.file_payloads
 
-    def create_payload(self) -> CreateEtchPacketPayload:
+    def get_files_for_payload(self) -> Tuple[List[AttachableEtchFile], List[Any]]:
```

Comment:
This also feels like something that belongs in a more shared, non-CreateEtchPacket specific spot.

This seem seems akin to https://www.npmjs.com/package/extract-files

Our node-anvil library has a decent pattern to follow where it figures out if there files to map [here](https://github.com/anvilco/node-anvil/blob/master/src/index.js#L409) and then makes a multi-part request if necessary [here](https://github.com/anvilco/node-anvil/blob/master/src/index.js#L418).

Reply:
The general `extract-files`-like functionality is in the function here: https://github.com/anvilco/python-anvil/pull/41/files#diff-615974429dd7f7e34ce2e08e0012206ae6f47d86bcfc9338e6f72546e76436beR14

In this first pass, though, this is mainly to add multipart support on `createEtchPacket` since it's the only one that has underlying model support. Will have a more generic multipart mutation path after this PR.